libcxx/test/libcxx/odr_signature.hardening.sh.cpp

```diff
@@ -0,0 +1,63 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Do we want to also check that the library version is encoded? (Maybe not, since it's not supposed to be overridden by users)

Reply:
I thought about that, but testing this from within the test suite is really challenging -- we'd have to get two different versions of libc++ or something like that. I think the current tests strike the appropriate balance but I'm open to being convinced otherwise.